tests/unit/AnnualReport.spec.ts

```diff
@@ -28,23 +32,27 @@ Vue.use(Vuetify)
 Vue.use(Vuelidate)
 
 const vuetify = new Vuetify({})
-const store = getVuexStore() as any // remove typings for unit tests
+setActivePinia(createPinia())
+const businessStore = useBusinessStore()
+const configurationStore = useConfigurationStore()
+const rootStore = useRootStore()
 
 describe('Annual Report - Part 1 - UI', () => {
   beforeEach(() => {
     // init store
-    store.commit('setIdentifier', 'CP0001191')
-    store.state.ARFilingYear = 2017
-    store.state.arMinDate = '2017-01-01'
-    store.state.arMaxDate = '2018-04-30'
-    store.state.currentFilingStatus = 'NEW'
-    store.commit('setLegalType', 'CP')
-    store.commit('setTestConfiguration', { key: 'VUE_APP_PAY_API_URL', value: 'https://auth.web.url/' })
+    businessStore.setIdentifier('CP0001191')
+    rootStore.ARFilingYear = 2017
+    rootStore.arMinDate = '2017-01-01'
+    rootStore.arMaxDate = '2018-04-30'
+    rootStore.currentFilingStatus = FilingStatus.NEW
+    businessStore.setLegalType(CorpTypeCd.COOP)
+    configurationStore.setTestConfiguration({ configuration: null },
```

Comment:
Yes, this does look strange to me. Does it work in localhost, just not in the tests?

Reply:
Yes. I went ahead and tried to file an Annual Report as a regular user. It was taking me to the credit card payment page. So, hopefully this should be OK although the test is still not working 😞

Reply:
That's a relief.

OK, lmk the new ticket number and I'll try to grab it soon.

Reply:
Is there a new ticket for this?

Reply:
New ticket: https://github.com/bcgov/entity/issues/16392